huaweicloud/services/acceptance/vpcep/resource_huaweicloud_vpcep_service_test.go

```diff
@@ -59,7 +60,8 @@ func TestAccVPCEPService_Basic(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "port_mapping.0.protocol", "TCP"),
 					resource.TestCheckResourceAttr(resourceName, "port_mapping.0.service_port", "8088"),
 					resource.TestCheckResourceAttr(resourceName, "port_mapping.0.terminal_port", "80"),
-					resource.TestCheckResourceAttr(resourceName, "permissions.#", "1"),
+					resource.TestCheckResourceAttr(resourceName, "permissions.0", "*"),
+					resource.TestCheckResourceAttr(resourceName, "organization_permissions.0", "organizations:orgPath::*"),
```

Comment:
Why using "*" for all domains, but using "organizations:orgPath::*0" for all organizations?